examples/digital_fingerprinting/production/morpheus/dfp/modules/dfp_pipeline_preprocessing.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+import dfp.modules.dfp_preprocessing  # noqa: F401
+import dfp.modules.dfp_rolling_window  # noqa: F401
+import dfp.modules.dfp_split_users  # noqa: F401
+import dfp.modules.dfp_training  # noqa: F401
+import mrc
+
+import morpheus.modules.file_batcher  # noqa: F401
+import morpheus.modules.file_to_df  # noqa: F401
+from morpheus.utils.module_utils import get_module_config
+from morpheus.utils.module_utils import load_module
+from morpheus.utils.module_utils import register_module
+
+logger = logging.getLogger(f"morpheus.{__name__}")
+
+
+@register_module("DFPPipelinePreprocessing", "morpheus_modules")
+def dfp_pipeline_preprocessing(builder: mrc.Builder):
+    """
+    This module function allows for the consolidation of multiple dfp pipeline preprocessing modules
+    into a single module.
+
+    Parameters
+    ----------
+    builder : mrc.Builder
+        Pipeline budler instance.
+    """
+
+    module_id = "DFPPipelinePreprocessing"
```

Comment:
For all strings like this where they are used multiple times, let's store the value in a variable